src/main/java/hudson/plugins/sonar/utils/Version.java

```diff
@@ -55,14 +56,17 @@ private void parse() {
   public final String get() {
     return this.versionStr;
   }
-
+  
   @Override
   public String toString() {
     return this.versionStr;
   }
 
   @Override
   public int compareTo(Version that) {
+    if (that == null) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2583)